src/content.ts

```diff
@@ -2,7 +2,6 @@ import BlobAnimation from './blob_animation'
 import { getStorage, logIntentToStorage } from './storage'
 import { cleanDomain } from './util'
 import injectOptionsToOnboarding from './onboarding_options'
```

Comment:
Kind of curious what was SimpleMDE for?

Reply:
we were thinking about adding a markdown editor to the main block page for people to take notes but scraped that halfway LOL